### ISSUE TITLE
@RunWith runs my tests multiple times

### ISSUE THREAD

**Issue by RajAsapu** [junit-team/junit4#1459]
I am using arquillian for running my docker containers and i am using @RunWith to trigger the arquillian and the cucumber tests. All my tests run 4 times instead of one time. Below is my code

Environment:
Junit 4.12
CukeSpace 1.6.3
Arquillian Cube 1.1.0

Code:
```
`import com.google.common.base.Verify;
import cucumber.api.CucumberOptions;
import cucumber.api.java.After;
import cucumber.runtime.arquillian.CukeSpace;
import net.masterthought.cucumber.Configuration;
import net.masterthought.cucumber.ReportBuilder;
import net.masterthought.cucumber.Reportable;
import org.arquillian.cube.CubeIp;
import org.arquillian.cube.HostPort;
import org.arquillian.cube.docker.impl.requirement.RequiresDocker;
import org.jboss.arquillian.container.test.api.RunAsClient;
import org.junit.Before;
import org.junit.runner.RunWith;
import setup.UpdateProperties;
import java.io.File;
import java.lang.annotation.Repeatable;
import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;

@RunWith(CukeSpace.class)
@CucumberOptions(
        strict = true,
        plugin = {"html:target/cucumber-html-report"},
        features = {"src/test/resources/features/pageobjects/"},
        glue = {"classpath:"},
        tags = {"@TestData"}
)
@RunAsClient
public class RunTest {

    @CubeIp(containerName = "database")
    protected String ipDatabase;
    @Before
    public void initialization() throws Exception {
            Verify.verify(props.startServiceContainer(ipDatabase, "config:latest"));
            System.out.println("Arquillian - Containers has started .");
        }
    }
}`
```

Note: All my tests including the before statement run 4 times

**Comment by panchenko** [comment]
Looking on how often `junit` happens in your code I think you would have more luck asking this question on the _arquillian_ or _cucumber_ resource.

**Comment by RajAsapu** [comment]
@panchenko : did that but no response yet

**Comment by RajAsapu** [comment]
Got resolved once i upgraded cukespace to 1.6.7
